### PR TITLE
Rewrite how to use page, and update contributing page

### PR DESCRIPTION
**Pull request by kmoscoe**
This PR rewrites the "How to use DC" page to incorporate the suggestions from Prashanth. Also fixes some outdated info on the "Contributing" page.

Staged at http://bullie.svl.corp.google.com:4000/how_to_use.html

### CODE REVIEW

File: how_to_use.md [hunk 1]
```diff
@@ -4,51 +4,59 @@ title: How to use Data Commons
...
+For quick analysis, use the search query bar on the [home page](https://datacommons.org/) or use any of the visualization tools, such as the [Timeline](https://datacommons.org/tools/visualization#visType=timeline), [Scatter](https://datacommons.org/togitols/visualization#visType%3Dscatter), and [Map](https://datacommons.org/tools/visualization#visType%3Dmap) explorers.
 
-If you want to issue SQL queries, and you have a Google Cloud Platform account, use BigQuery Studio on Data Commons data in [Analytics Hub](https://cloud.google.com/analytics-hub). See the [Data Commons in BigQuery](/bigquery/index.html) page for more details.
+## Learn about the data in Data Commons {#learn}
 
-## Issue programmatic data queries
+To find out what data is available in Data Commons, check out the [Statistical Variable Explorer](https://datacommons.org/tools/statvar) and see the [Data sources](/datasets) pages.
```
Comment (pradh) [inline how_to_use.md:27]: Wonder if Statistical Variable might be a foreign concept and if its worth linking to (or having a sentence with) its definition?

File: how_to_use.md [hunk 2]
```diff
@@ -4,51 +4,59 @@ title: How to use Data Commons
...
 
-Data Commons publishes REST, Python, Pandas, Google Sheets, and SPARQL [APIs](/api/index.html). The APIs support both low-level exploration of the knowledge graph as well as higher-level statistical analysis of variables.
+## Query the Data Commons data directly {#query}
 
-The Python and pandas APIs provide convenient wrappers for calling the APIs; we have developed a set of [Google Colab tutorials](/tutorials/index.html) to help you get started with analysis. We have also developed a [Data science curriculum](/courseware/intro_data_science.html) featuring our API and data, currently in use at MIT.
+There are several options for directly querying the data, without accessing the datacommons.org website, both interactive and programmatic:
```
Comment (pradh) [inline how_to_use.md:32]: nit: Maybe a paragraph break between REST APIs and Python APIs?

**Review by pradh — APPROVED** [review]
Very nice!

File: how_to_use.md (hunk 1 above)
Comment (kmoscoe) [reply how_to_use.md]: Let's skip that since there's a dedicated doc about it now.

File: how_to_use.md (hunk 2 above)
Comment (kmoscoe) [reply how_to_use.md:32]: Done.

**Pull request merged**